routes/teamAPI.js

```diff
@@ -11,26 +11,34 @@ var router = require('express').Router();
 //		coach:String (user)
 //		lead:String(user)
 //		member:[String] (user)
-//		project: [String] (project) 暂时未开放
+//		project: [String] (project)
+//response:
+// {team:{TeamEntity}}
 router.post('/add',authenticator.authenticate,teamController.create);
 
 //query parameters
 //	required:
 //		id:String (idea)
+//response:
+// {team:{TeamEntity}}
 router.get('/:id',teamController.getTeamById);
 
 //body parameters
 //	required:
 //	optional:
-//		pageNum:Number (default:1)
+//		pageNum:Number (default:0)
 //		pageSize:Number (default:10)
 //		keyword:String (for name)
+//response:
+// {total:totalNumber,teams:[TeamEntities]}
 router.post('/list',teamController.list);
 
 
-//query parameters
+//body parameters
 //	required:
 //		name:String 
+//response:
+//{exist:true/false}
 router.post('/check/name',teamController.checkNameExist);
```

Comment:
I remember I had this comment before, why do you need the second slash? Also why making it a post method. In this case, get method is more fit. It is a purely get action.   Please adhere to http/REST standards recommendation as much as possible.